middleware.ts

```diff
@@ -0,0 +1,60 @@
+import type { NextRequest } from "next/server";
+import { geolocation, ipAddress } from "@vercel/edge";
+import { NextResponse } from "next/server";
+
+const BLOCKED_COUNTRIES = [
+  "CU", // Cuba
+  "IR", // Iran
+  "KP", // North Korea
+  "SY", // Syria
+  "RU", // Russia
+  "unknown", // Block if we cannot identify country
```

Comment:
Same here for country. Blocking by default if we cannot detect the country. We do have server side logs to see if we do hit this case.

Reply:
cool, would be good if we can test this to make sure Vercel's geo locating is quite reliable and we don't end up blocking a majority of users due to this fallback.